data-url/src/mime.rs

```diff
@@ -12,6 +12,20 @@ pub struct Mime {
 }
 
 impl Mime {
+    pub fn new(type_: &str, subtype: &str) -> Self {
```

Comment:
A line of docs here? Just mention that this constructs the MIME `<type>/<subtype>` with no parameters

Reply:
Sounds good. I've added:

```diff
+    /// Construct a new [`Mime`] with the given `type_` and `subtype` and an
+    /// empty parameter list.
     pub fn new(type_: &str, subtype
```